packages/auth/src/api/index.ts

```diff
@@ -198,7 +199,9 @@ export async function _performFetchWithErrorHandling<V>(
   customErrorMap: Partial<ServerErrorMap<ServerError>>,
   fetchFn: () => Promise<Response>
 ): Promise<V> {
-  (auth as AuthInternal)._canInitEmulator = false;
+  const authInternal = auth as AuthInternal;
+  updateEmulatorBanner('Auth', authInternal.emulatorConfig !== null);
```

Comment:
do we need to update this on every call? or just when we do the ping?

Reply:
What about when the user connects to prod?